controllers/dashboard_test.go

```diff
@@ -163,6 +163,35 @@ var _ = Describe("DashboardController", func() {
 					Expect(len(matches)).To(BeZero())
 				})
 			})
+
+			Context("given `keyword` param is set", func() {
+				It("shows only `keyword` filter records", func() {
```

Comment:
To be correct syntaxically, it should be:

```suggestion
				It("shows only `keyword` filtered records", func() {
```